include/ginkgo/core/solver/idr.hpp

```diff
@@ -251,7 +251,8 @@ class Idr : public EnableLinOp<Idr<ValueType>>,
                     LinOp *x) const override;
 
     template <typename SubspaceType>
-    void iterate(const LinOp *b, LinOp *x) const;
+    void iterate(const matrix::Dense<SubspaceType> *dense_b,
+                 matrix::Dense<SubspaceType> *dense_x) const;
```

Comment:
Technically that's an interface break? Though at least IDR was not released yet.

Reply:
Good point, I thought that member function was private.

Reply:
As you mention, as IDR is not released, I also think it should be okay. Even if spack or someone else pulls develop, we never guarantee that develop has a consistent interface in itself.